specification/schemas/ServiceRate.json

```diff
@@ -15,9 +15,41 @@
             },
             "weight_max": {
               "type": "integer",
-              "example": 2000,
+              "example": 20000,
               "description": "Weight in grams."
             },
+            "weight_bracket": {
+              "type": "object",
+              "required": [
+                "start",
+                "start_amount",
+                "size",
+                "size_amount"
+              ],
+              "properties": {
+                "start": {
+                  "type": "integer",
+                  "example": 6000,
+                  "description": "Weight in grams."
+                },
+                "start_amount": {
+                  "type": "integer",
+                  "example": 500,
+                  "description": "The starting amount of the bracket price in cents/pence."
+                },
+                "size": {
+                  "type": "integer",
+                  "example": 2000,
+                  "description": "Weight in grams."
+                },
```

Comment:
This should be a bit more informative. What does `size` mean? It's weight in grams, but what does it represent? It took me a while to figure that out, I think our users might even take longer. 

Let's add some information to what it represents, like calling it `bracket_size`, or at least adding info to the description.